src/components/dashboard/orders/Order.js

```diff
@@ -0,0 +1,126 @@
+import React, {Component, PropTypes} from 'react';
+import ReactDataGrid from 'react-data-grid';
+// import {update} from 'react-addons-update';
+// import {Editors, Formatters} from 'react-data-grid-addons';
+
+import OrderEdit from './OrderEdit';
+import BooleanFormatter from './BooleanFormatter';
+import ArrayFormatter from './ArrayFormatter';
+
+class Order extends Component {
+    constructor(props) {
+        super(props);
+
+       this._columns = [{
+            key: 'id',
```

Comment:
we don't need `orderId` this value doesn't make sense to users. We need the name of the item that the order references, the amount, and the date the order was created